packages/moonstone/ProgressBar/ProgressBarTooltip.js

```diff
@@ -1,12 +1,34 @@
 import kind from '@enact/core/kind';
+import {memoize} from '@enact/core/util';
+import ilib from '@enact/i18n';
 import {contextTypes} from '@enact/i18n/I18nDecorator';
+import NumFmt from '@enact/i18n/ilib/lib/NumFmt';
 import React from 'react';
 import PropTypes from 'prop-types';
+import warning from 'warning';
 
 import Tooltip from '../TooltipDecorator/Tooltip';
 
 import css from './ProgressBarTooltip.less';
 
+const memoizedPercentFormatter = memoize((/* locale */) => new NumFmt({
+	type: 'percentage',
+	useNative: false
+}));
+
+const getSide = (orientation, side) => {
```

Comment:
Should we also have a guard like `getOrientation` for `orientation` as well? When `orientation` gets  a non recognizable value, the tooltip doesn't position correctly.

Reply:
`PropTypes` should be a sufficient guard for that. In that light, we could (should?) move this logic into a custom PropType function instead which would make them consistent.